<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>牵引供电系统计算软件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveas"/>
    <addaction name="separator"/>
    <addaction name="actionExamples"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="action_source_lines"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>结构配置</string>
    </property>
    <addaction name="actionSetlines"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>计算参数配置</string>
    </property>
    <addaction name="actionpeizhi"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>结果输出</string>
    </property>
    <addaction name="actionshuchu"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="statusTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="statusTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="statusTip">
    <string>保存线路设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveas">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="actionExamples">
   <property name="text">
    <string>示例(&amp;E）</string>
   </property>
   <property name="statusTip">
    <string>线路示例</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>打印(&amp;P）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionLinespara">
   <property name="text">
    <string>设置</string>
   </property>
   <property name="statusTip">
    <string>内部参数设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionHlink">
   <property name="text">
    <string>横向连接元件</string>
   </property>
   <property name="statusTip">
    <string>横向连接元件参数设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionLinespara_2">
   <property name="text">
    <string>导线参数</string>
   </property>
   <property name="statusTip">
    <string>导线参数设置</string>
   </property>
  </action>
  <action name="actionHlink_2">
   <property name="text">
    <string>横向连接元件参数</string>
   </property>
   <property name="statusTip">
    <string>横向连接元件参数设置</string>
   </property>
  </action>
  <action name="actionSetlines">
   <property name="text">
    <string>线路配置</string>
   </property>
  </action>
  <action name="actionLinespara_4">
   <property name="text">
    <string>导线参数</string>
   </property>
  </action>
  <action name="actionElementpara">
   <property name="text">
    <string>横向元件参数</string>
   </property>
  </action>
  <action name="actionM">
   <property name="text">
    <string>大矩阵M</string>
   </property>
  </action>
  <action name="actionUI">
   <property name="text">
    <string>节点电压电流U、I</string>
   </property>
  </action>
  <action name="actionZ">
   <property name="text">
    <string>短路阻抗Z</string>
   </property>
  </action>
  <action name="actionXiebo">
   <property name="text">
    <string>各次谐波</string>
   </property>
  </action>
  <action name="actionDuodao">
   <property name="text">
    <string>多导体阻抗矩阵</string>
   </property>
  </action>
  <action name="actionDaorong">
   <property name="text">
    <string>电容导纳</string>
   </property>
  </action>
  <action name="actionZhubianyaqi">
   <property name="text">
    <string>主变压器导纳矩阵</string>
   </property>
  </action>
  <action name="actionATdaona">
   <property name="text">
    <string>AT导纳矩阵</string>
   </property>
  </action>
  <action name="actionHengxianglianj">
   <property name="text">
    <string>横向连接线矩阵</string>
   </property>
  </action>
  <action name="actionGongdianbi_fenduan">
   <property name="text">
    <string>供电臂分段</string>
   </property>
  </action>
  <action name="actionF">
   <property name="text">
    <string>负荷（机车）导纳矩阵、电流矩阵</string>
   </property>
  </action>
  <action name="actionF_2">
   <property name="text">
    <string>故障（短路）导纳矩阵</string>
   </property>
  </action>
  <action name="actionF_3">
   <property name="text">
    <string>链式网络模型</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>接触网电压曲线</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>接触网电流曲线</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>钢轨电位曲线</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>大地回流曲线</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>短路阻抗曲线</string>
   </property>
  </action>
  <action name="action6_2">
   <property name="text">
    <string>谐波电压分布曲线</string>
   </property>
  </action>
  <action name="action7">
   <property name="text">
    <string>谐波电流分布曲线</string>
   </property>
  </action>
  <action name="action_source_lines">
   <property name="text">
    <string>原始导线参数管理</string>
   </property>
  </action>
  <action name="actionpeizhi">
   <property name="text">
    <string>配置</string>
   </property>
  </action>
  <action name="actionshuchu">
   <property name="text">
    <string>输出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
